### PR TITLE
Update AusTraits tutorial

### PR DESCRIPTION
**Pull request by ehwenk**
- update austraits.build tutorial, following release of austraits.build v5.0.0
- minor updates to other traits.build-book files
- the traits-climate example does not evaluate online, as we've decided for now not to host a copy of the WorldClim file

### CODE REVIEW

File: spatial_data_example.qmd
Comment (dfalster) [inline spatial_data_example.qmd:8]: You'll want to change this as soon as Fonti has merged in the branch

File: AusTraits_tutorial.qmd
Comment (dfalster) [inline AusTraits_tutorial.qmd:17]: Should we point to the package or the book?

File: AusTraits_tutorial.qmd
Comment (dfalster) [inline AusTraits_tutorial.qmd:556]: elsewhere you've been writing dplyr::filter

File: AusTraits_tutorial.qmd
Comment (dfalster) [inline AusTraits_tutorial.qmd:567]: and here

File: AusTraits_tutorial.qmd
Comment (dfalster) [inline AusTraits_tutorial.qmd:580]: and here

File: AusTraits_tutorial.qmd
Comment (dfalster) [inline AusTraits_tutorial.qmd:589]: and here

File: AusTraits_tutorial.qmd
Comment (dfalster) [inline AusTraits_tutorial.qmd:604]: ok, getting repetitive

**Review by dfalster — APPROVED** [review]

File: AusTraits_tutorial.qmd
Comment (ehwenk) [reply AusTraits_tutorial.qmd:17]: I've rewritten to first point to book, then below mention there are individual GitHub repos

File: AusTraits_tutorial.qmd
Comment (ehwenk) [reply AusTraits_tutorial.qmd:556]: fixed

**Comment by ehwenk** [comment]
@dfalster As you've pointed out, I'll wait to merge this in until after Fonti has merged in the `{austraits}` update.

File: AusTraits_tutorial.qmd
Comment (yangsophieee) [inline AusTraits_tutorial.qmd:45]: Should this be [[3]]?

Also up to you but could swap to tidyverse version of code for readability:
```austraits::get_versions() |> dplyr::pull("version") |> dplyr::first()```

File: AusTraits_tutorial.qmd
Comment (yangsophieee) [inline AusTraits_tutorial.qmd:152]: What do the backslashes do?

File: AusTraits_tutorial.qmd
Comment (yangsophieee) [inline AusTraits_tutorial.qmd:195]: Is it `plyr::join` not `dplyr::join`? That's what comes up with the help file

File: AusTraits_tutorial.qmd
Comment (yangsophieee) [inline AusTraits_tutorial.qmd:350]: Typo apostrophes

**Review by yangsophieee — APPROVED** [review]
I've only skimmed the code and can't render the book/Quarto files as my `austraits` (updated to the v5 branch) doesn't seem to like the version "5.0.0"? 

This is a huge amount of work, great job Lizzy!

**Comment by yangsophieee** [comment]
@ehwenk I figured out why I couldn't render. I will try render now but it's because of `load_austraits(doi = most_recent)`, should be `version = most_recent` (see comment)

File: AusTraits_tutorial.qmd
Comment (ehwenk) [reply AusTraits_tutorial.qmd:48]: Good idea - but the [[2]] is correct, because I'm loading from the doi. Partially so people see that there is a doi for the version

File: AusTraits_tutorial.qmd
Comment (ehwenk) [reply AusTraits_tutorial.qmd:155]: They are a forced line breaks  and also "escape" codes added before certain characters - and in some circumstances they automatically add themselves

File: AusTraits_tutorial.qmd
Comment (ehwenk) [reply AusTraits_tutorial.qmd:198]: I guess I was referring to the family of join functions in dplyr.

File: AusTraits_tutorial.qmd
Comment (ehwenk) [reply AusTraits_tutorial.qmd:353]: I just changed to "you will"

**Pull request merged**
